breaking-changes-2.0.md

```diff
@@ -0,0 +1,6 @@
+# List of breaking changes for recheck 2.0
```

Comment:
Wouldn' t this be best put into the wiki?

Reply:
Do we need this list at all? I would suggest to use issues and create a label like `recheck 2.0.0`. Our `@Deprecated` markers should help here.

Reply:
Ok. I'll remove this from the PR and we can solve it separately...